nvflare/job_config/script_runner.py

```diff
@@ -45,8 +45,7 @@ def __init__(
         framework: FrameworkType = FrameworkType.PYTORCH,
         params_transfer_type: str = TransferType.FULL,
         executor: Union[ClientAPILauncherExecutor, InProcessClientAPIExecutor, None] = None,
-        from_nvflare_converter_id: Optional[str] = None,
```

Comment:
Should we still keep these? If the user provides converter_ids, we should use them. Otherwise, use the logic based on params_exchange_format.

Reply:
right, let's add them back to base runner